framework/src/fvbcs/FVBoundaryCondition.C

```diff
@@ -0,0 +1,70 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "FVBoundaryCondition.h"
+#include "Problem.h"
+#include "SystemBase.h"
+#include "MooseVariableFV.h"
+
+InputParameters
+FVBoundaryCondition::validParams()
+{
+  InputParameters params = MooseObject::validParams();
+  params += TransientInterface::validParams();
+  params += BoundaryRestrictableRequired::validParams();
+  params += TaggingInterface::validParams();
+
+  params.addRequiredParam<NonlinearVariableName>(
+      "variable", "The name of the variable that this boundary condition applies to");
+  params.addParam<bool>("use_displaced_mesh",
+                        false,
+                        "Whether or not this object should use the "
+                        "displaced mesh for computation.  Note that "
+                        "in the case this is true but no "
+                        "displacements are provided in the Mesh block "
+                        "the undisplaced mesh will still be used.");
```

Comment:
should we error if someone sets this to `true` for now?

Reply:
There is a check+error in the FVKernel constructor, just missed adding one here.

Reply:
fixed.